conf/examples/acm_hub_unreleased_image.yaml

```diff
@@ -1,3 +1,3 @@
 ---
 ENV_DATA:
-  acm_unreleased_image: "2.5.2-DOWNSTREAM-2022-07-29-15-56-49"
+  acm_unreleased_image: "2.7.0-DOWNSTREAM-2023-01-07-13-58-25"
```

Comment:
update image to 2.7.0-DOWNSTREAM-2023-01-26-20-15-10

Reply:
@prsurve can we merge and then update this image, I am worried if it breaks again then PR merge will be further pushed. WDYT?

Reply:
ack that should work

Reply:
> ack that should work

Thanks Pratik